Packs/FeedCrowdstrikeFalconIntel/ReleaseNotes/2_0_1.md

```diff
@@ -0,0 +1,4 @@
+#### Integrations
+##### CrowdStrike Falcon Intel Feed Actors
+- Maintenance and stability enhancements.
```

Comment:
replace this with added support for reset last run for Cortex XSOAR version 6.2

Reply:
done